README.md

```diff
@@ -4,79 +4,136 @@
 
 Inspired by Eric Meyers CSS Reset, Git New is a simple tool to enforce a consistent baseline Git experience for new projects.
 
-## Installation
+- [Git New](#git-new)
 
-Linux, Windows and OSX
+  - [For Project Developers](#for-project-developers)
+    - [Prerequisites](#prerequisites)
+    - [Setup and Configuration](#setup-and-configuration)
+  - [For Installers](#for-installers)
+    - [Prerequisites](#prerequisites-1)
+    - [Setup and Configuration](#setup-and-configuration-1)
+  - [Change log](#change-log)
+  - [Meta](#meta)
+  - [Contributing](#contributing)
+
+## For Developers
+
+Follow this section if you are developing with, installing or deploying this repository.
+
+### Prerequisites
+
+- A good code editor. We suggest [VSCode](https://code.visualstudio.com/), [Atom](https://atom.io/) or [SublimeText](https://www.sublimetext.com/).
+- [NodeJS LTS+ and NPM 6.14+](https://nodejs.org/en/download/)
```

Comment:
Should we specify versions for things like Node to future proof it? I don't know if patch or even minor is necessary, but I would think major might be handy, as things could change down the road. Thoughts?

Reply:
What's your view on the LTS convention? When I work (assuming I am not constrained by AWS Lambda runtimes) I tend to go for the latest Node version. And I will live with the consequences of being on the breaking edge. But for collab/shared source should we stick with latest and just list the version? Or favour stability and recommend LTS?

Reply:
I like favouring the current LTS for shared source or client projects.  However, simply listing LTS doesn't help when we try to run the project a few years later as there could be a different LTS version.  In the future, if I knew it was a specific major (even minor) version, I could use NVM to switch before running it.  This mostly applies to projects that come and go quickly and don't see long term development, where the code base would evolve as the tools do.

Reply:
However, if we pin to a version (use Node as an example) and someone clones the repo 2 years later does that imply they have to downgrade their version to one from 2 years prior? 

I was adding a plus sign (e.g. NPM 6.14+) but that gets a bit redundant. 

```suggestion
### Prerequisites

The versions listed for these prerequisites are current at the time of writing. More recent versions will likely work but "your mileage may vary".

- A good code editor. 
- [Node.js v12.13.0 and NPM 6.14.5](https://nodejs.org/en/download/)
```

Reply:
FYI, I'm using ctrl-g in GitHub to insert suggested changes that can be accepted and merged from this page (it's pretty cool). But in the above one I notice it is not replacing the full text, only the line :(

Reply:
I think it depends on what changes over time, which is out of our control.  Pinning to a major or even minor version may guarantee that it will run on that version, where future major versions may intro breaking changes that is incompatible with our code.  Most of our projects will be fine running in the future, but there is a chance that it doesn't run on Node 20.  I think adding that line that you did makes the most sense.

Reply:
I see what you mean, it duplicated the content in the commit.